### PR TITLE
Add default configs

### LINKED ISSUE #52: Update default config files for different platforms
The default config should enable all ports with max speed, for example:
Wedge100 BF 32x: 32x100G ports

The current default config file creates 0 ports when Stratum started.

### PR DESCRIPTION
**Pull request by Yi-Tseng**
Add chassis and PHAL configs for:
 - Accton AS7712
 - Accton Wedge100BF-32X
 - Accton Wedge100BF-65X
 - Dell Z9100
 - Delta AG9064v1
 - Inventec D5254
 - Quanta IX1 Rangeley
 - Stordis BF2556X
 - Stordis BF6064X

And default BCM chassis maps for:
 - Accton AS7712
 - Dell Z9100
 - Inventec D7032Q
 - Quanta IX1 Rangeley

Fixes #52

### CODE REVIEW

File: stratum/hal/config/BUILD [hunk 1]
```diff
@@ -30,42 +30,9 @@ package(
...
-        "base_bcm_chassis_map_generic_trident2_hercules.pb.txt",
-        "base_bcm_chassis_map_generic_trident2_hercules.pb.txt",
-        "bcm_hardware_specs.pb.txt",
-        "generic_trident2_bcm_serdes_db_hercules.pb.bin",
-        "generic_trident2_bcm_serdes_db_hercules.pb.bin",
-    ],
-)
+        "bcm_hardware_specs.pb.txt"
+    ]
+)
```
Comment (pudelkoM) [inline stratum/hal/config/BUILD]: Newline

File: stratum/hal/config/config_validator/main.cc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+#endif
+
+}  // namespace hal
+}  // namespace stratum
+
+int main(int argc, char** argv) {
+  ::testing::InitGoogleTest(&argc, argv);
+  return RUN_ALL_TESTS();
+}
```
Comment (pudelkoM) [inline stratum/hal/config/config_validator/main.cc]: A `main` should not be needed there is a `gtest_main` target for the default case.

File: stratum/hal/config/x86-64-accton-as7712-32x-r0/BUILD [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        "@com_github_grpc_grpc//:grpc++",
+    ],
+    data = [
+        ":configs"
+    ],
+    defines = [
+        "PLATFORM=x86-64-accton-as7712-32x-r0",
+        "BCM_TARGET",
+    ]
+)
```
Comment (pudelkoM) [inline stratum/hal/config/x86-64-accton-as7712-32x-r0/BUILD]: newline

File: stratum/hal/config/x86-64-accton-wedge100bf-32x-r0/BUILD [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        "@com_google_googletest//:gtest",
+        "@com_github_grpc_grpc//:grpc++",
+    ],
+    data = [
+        ":configs"
+    ],
+    defines = [
+        "PLATFORM=x86-64-accton-wedge100bf-32x-r0",
+    ]
+)
```
Comment (pudelkoM) [inline stratum/hal/config/x86-64-accton-wedge100bf-32x-r0/BUILD]: newline

File: stratum/hal/config/x86-64-accton-wedge100bf-65x-r0/BUILD [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        "@com_google_googletest//:gtest",
+        "@com_github_grpc_grpc//:grpc++",
+    ],
+    data = [
+        ":configs"
+    ],
+    defines = [
+        "PLATFORM=x86-64-accton-wedge100bf-65x-r0",
+    ]
+)
```
Comment (pudelkoM) [inline stratum/hal/config/x86-64-accton-wedge100bf-65x-r0/BUILD]: newline

File: stratum/hal/config/x86-64-dell-z9100-c2538-r0/BUILD [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        "@com_github_grpc_grpc//:grpc++",
+    ],
+    data = [
+        ":configs"
+    ],
+    defines = [
+        "PLATFORM=x86-64-dell-z9100-c2538-r0",
+        "BCM_TARGET",
+    ]
+)
```
Comment (pudelkoM) [inline stratum/hal/config/x86-64-dell-z9100-c2538-r0/BUILD]: newline

File: stratum/hal/config/config_validator/main.cc (hunk 2 above)
Comment (Yi-Tseng) [reply stratum/hal/config/config_validator/main.cc]: done

**Comment by Yi-Tseng** [comment]
> Missing newlines on all files.
> 
> The BUILD file is pretty much 100% duplicated across the folders. Is it possible to have one BUILD files in `stratum/hal/configs/` that handles all configs?

Agree, we can define a function that creates rules for all targets (see new BUILD file)
https://github.com/stratum/stratum/pull/58/files#diff-4a1981cbd9148ade0410403234d53095R37-R47

**Pull request merged**